src/connectors/rating-menu/__tests__/connectRatingMenu-test.js

```diff
@@ -248,6 +253,95 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/rating-menu
     }
   });
 
+  describe('getConfiguration', () => {
+    test('returns initial search parameters', () => {
+      const rendering = jest.fn();
+      const makeWidget = connectRatingMenu(rendering);
+
+      const attribute = 'grade';
+      const widget = makeWidget({
+        attribute,
+      });
+
+      expect(widget.getConfiguration(new SearchParameters({}))).toEqual(
+        new SearchParameters({
+          disjunctiveFacets: [attribute],
+          disjunctiveFacetsRefinements: {
+            grade: [],
+          },
+        })
+      );
+    });
+
+    test('supports previous disjunctive facets refinements', () => {
+      const rendering = jest.fn();
+      const makeWidget = connectRatingMenu(rendering);
+
+      const attribute = 'grade';
+      const widget = makeWidget({
+        attribute,
+      });
+
+      expect(
+        widget.getConfiguration(
+          new SearchParameters({
+            disjunctiveFacets: [attribute],
+            disjunctiveFacetsRefinements: {
+              grade: [4],
+            },
+          })
+        )
+      ).toEqual(
+        new SearchParameters({
+          disjunctiveFacets: [attribute],
+          disjunctiveFacetsRefinements: {
+            grade: [4],
+          },
+        })
+      );
+    });
+  });
+
+  describe('dispose', () => {
+    test('calls the unmount function', () => {
+      const render = jest.fn();
+      const unmount = jest.fn();
+      const makeWidget = connectRatingMenu(render, unmount);
+      const helper = jsHelper({}, '', {});
+      helper.search = jest.fn();
+
+      const attribute = 'grade';
+      const widget = makeWidget({
+        attribute,
+      });
+
+      widget.dispose({ state: helper.state });
+
+      expect(unmount).toHaveBeenCalledTimes(1);
+    });
+
+    test('resets the state', () => {
+      const render = jest.fn();
+      const makeWidget = connectRatingMenu(render);
+      const indexName = 'indexName';
+      const helper = jsHelper({}, indexName, {});
+      helper.search = jest.fn();
+
+      const attribute = 'grade';
+      const widget = makeWidget({
+        attribute,
+      });
+
+      const nextState = widget.dispose({ state: helper.state });
```

Comment:
It looks like the previous state does not contain `disjunctiveFacets` neither `disjunctiveFacetsRefinements`. It means that we don't actually test that the `dispose` function removes the refinements. We should apply refinements to the state then call the function.